### ISSUE TITLE
[BUG]: AccessibilityChecker raises violations for tabbable elements with the "application" role

### ISSUE THREAD

**Issue by Mark-Falconbridge-i2** [IBMa/equal-access#1183]
### Project

a11y checker engine, a11y checker

### Browser

Chrome

### Operating system

Windows

### Description

I have a web app which contains a Leaflet Map within it. That map has a tabIndex of 0 applied to it because the user can make the map scroll around using the arrow keys. I have applied the role of `application` to that element as that seems to be the best fit.

When I run the accessibility checker it reports a violation - "The tabbable elements's role 'application' is not a widget role".

According to the w3 spec, the `application` role seems to be the best fit for the map - https://www.w3.org/TR/wai-aria-1.2/#application. If the `application` role isn't the most appropriate, can you suggest an alternative?

Here is an image of the violation.
![MapWithApplicationRole](https://user-images.githubusercontent.com/97103678/205322308-a7700262-0ce9-4e61-9e7f-ddc0a975b39f.PNG)

### Steps to reproduce

1. Go to the example Leaflet map page here - https://leafletjs.com/examples/quick-start/example.html.
2. Edit the DOM using the Chrome dev tools to add `role="application"` to the element with the id "map".
3. Run the accessibility checker.
4. Notice the violation (amongst others) "The tabbable element's role 'application' is not a valid widget role".

**Comment by philljenkins** [comment]
Here is a screenshot before adding the `role=" application"` 
![image](https://user-images.githubusercontent.com/16193609/206571949-3cd57a61-e3e4-4b75-88f9-80d3ae09c656.png)

**Comment by philljenkins** [comment]
We're going to discuss this case. The ARIA 1.2 spec is somewhat unclear.  

@Mark-Falconbridge-i2 do  you have any other examples where you use `role="application"`

**Comment by Mark-Falconbridge-i2** [comment]
Hi Phil, No, I don't have any other examples.

**Comment by mbgower** [comment]
So s before digging into this...
1. If you are going to put the focus around the whole map in order to allow users to pan and reposition, I suggest you explore having icon appear on focus that indicators that, something like this or similar 
![image](https://user-images.githubusercontent.com/14298245/206787961-8a8b0445-f98f-4eec-a916-802004de7874.png)

ARIA doesn't really have a "pan" role, which is effectively what this interaction is. There are a few hits on [panning](https://www.w3.org/TR/SVG11/interact.html#ZoomAndPanAttribute) but they are all about panning within a magnified viewport. Too bad this concept isn't extended a bit more.

The interaction doesn't fit the [scrollbar](https://www.w3.org/TR/wai-aria-1.1/#scrollbar) role since there are no scrollbars visible (and potentially no top or bottom or end point horizontally) -- but scrolling inside the window is VERY similar to what's going on here.

I think putting a role of application on it seems fairly simple and effective. One could add in an aria-roledescription to help convey this "reposition map view" functionality, and possibly even aria-keyshortcuts to convey what they are.

But none of that solves the checker failure on using "application". You can take off the tabindex, at which point NO keyboard user can operate the panning, so that's hardly a good solution. 

I wonder if this error is appearing as a result of a tabindex rule?

**Comment by mbgower** [comment]
I tested with JAWS, and role="application" allows JAWS to pass through keys. JAWS also read out the aria-roledecription value, but not the keyshortcuts info. So I added that into the description like this:
aria-roledescription="Map view. Use arrow keys to reposition"

**Comment by aliunwala** [comment]
From triage:

We need to edit our rules to allow the role="application" to be tabbable. As the spec allows it here:
![image](https://user-images.githubusercontent.com/4451128/210885778-ad29fe6e-c34f-47d5-b135-c1dc640f3924.png)

https://www.w3.org/TR/wai-aria-1.2/#application

**Comment by sct-will** [comment]
Gated by build issue.
